GCTC-NTGC/gc-digital-talent
# Rename table experience_skills to experience_skill

Resolves #2403.

### Steps to Test
1. In a browser, view [experience_skills table structure with Adminer](http://localhost:8080/adminer?pgsql=postgres&username=postgres&db=gctalent&ns=public&table=experience_skills) and make note of the **Indexes** and **Foreign keys**
2. To run the new migration on the database from inside the [docker container](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/api/README.md#running-with-docker-containers), run `docker-compose exec -w /var/www/html/api php sh -c "php artisan migrate"`.
3. In a browser, view [experience_skill table structure with Adminer](http://localhost:8080/adminer?pgsql=postgres&username=postgres&db=gctalent&ns=public&table=experience_skill) and make note of the **Indexes** and **Foreign keys** (they should be the same as before the migration but any instances of _experience_skills_ will be _experience_skill_)
4. To reverse the migration on the database from inside the [docker container](https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/api/README.md#running-with-docker-containers), run `docker-compose exec -w /var/www/html/api php sh -c "php artisan migrate:rollback --step=1"`.
5. In a browser, view [experience_skills table structure with Adminer](http://localhost:8080/adminer?pgsql=postgres&username=postgres&db=gctalent&ns=public&table=experience_skills) and make note of the **Indexes** and **Foreign keys** (they should be the same as before the migration and the reversal of the migration with any instances of _experience_skill_ will be _experience_skills_)

### Important Note
For @tristan-orourke or whoever is releasing this to UAT and production: a backup of the database should be made before this migration is run (and then tested and verified) on UAT and production environments.
